packages/validator/src/cli-validator/utils/print-results.js

```diff
@@ -7,13 +7,12 @@ const each = require('lodash/each');
 const pad = require('pad');
 
 // this function prints all of the output
-module.exports = function print({ chalk, config, logger }, results) {
-  const { summaryOnly, errorsOnly } = config;
+module.exports = function print(context, results) {
+  const { chalk, config, logger } = context;
+  const types = ['error', 'warning', 'info', 'hint'];
```

Comment:
A added a few changes like this based on new use of config options and what seemed best for moving args around... don't mind me 🙂